tools/screenshot-test/src/app/firebase.service.ts

```diff
@@ -71,10 +71,7 @@ export class FirebaseService {
             this._readResults(childSnapshot);
             break;
         }
-        counter++;
-        if (counter === snapshot.numChildren()) {
-          return true;
-        }
+        return ++counter === snapshot.numChildren();
```

Comment:
Not sure. I thought it's pretty basic a pretty JavaScript syntax. I'm fine changing it if you prefer it though.